### PR TITLE
Added a bit more spacing (top) in list of episodes

### PR DESCRIPTION
**Pull request by sockeqwe**
Before:

before

After:
after

What do you think?

### CODE REVIEW

File: themes/context/layouts/_default/list.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h1>{{ .Title }}</h1>
 {{ .Content }}
 
 {{ range .Pages.ByDate.Reverse }}
-<h3><a href="{{ .URL }}">{{ .Title | markdownify }}</a></h3>
+<h3 class="episodelistTitle"><a href="{{ .URL }}">{{ .Title | markdownify }}</a></h3>
```
Comment (arturdryomov) [inline themes/context/layouts/_default/list.html]: Better make it an inline style since it is an ad-hoc solution. Or even use good old `<br/>` instead!

**Review by arturdryomov — CHANGES_REQUESTED** [review]
Also, change your Git email address. You skipped `r` in the last name.

File: themes/context/layouts/_default/list.html (hunk 1 above)
Comment (sockeqwe) [reply themes/context/layouts/_default/list.html]: done

**Review by arturdryomov — APPROVED** [review]

**Comment by sockeqwe** [comment]
Thanks! I regenerated website and published it ...

**Pull request merged**